### DISCUSSION TITLE
README 中的 LOGO 图片失效

### DISCUSSION THREAD

**Discussion by prinsss** [bs-community/blessing-skin-server#355] [General]
[README](https://github.com/bs-community/blessing-skin-server/blob/dev/README.md?plain=1#L4) 中的项目 LOGO 图片链接已经失效，可以考虑更换？将引用的图片放在仓库内而非外部图床应该是更好的选择。

P.S. 原始图片在[这里](https://img.prin.studio/images/2017/01/01/bs-logo.png)，供参考。

**Comment by g-plane** [comment]
谢谢。

**Comment by g-plane** [comment]
已修复。
